lib/fileutils.rb

```diff
@@ -1642,6 +1642,12 @@ def install(src, dest, mode: nil, owner: nil, group: nil, preserve: nil,
       st = File.stat(s)
       unless File.exist?(d) and compare_file(s, d)
         remove_file d, true
+        begin
+          require 'pathname'
+        rescue LoadError
+          raise "Cannot load module pathname."
```

Comment:
Yes, that was the wrong suggestion.  Can the input parameter for the directory be used instead?  I'm just thinking the current message is not as helpful for debugging.

Reply:
How does the new verbiage sound?

Reply:
That is a bit more clear to hep debug.

Reply:
As per @supechicken's change I removed the entire recovery block.